docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -42,47 +42,73 @@ Example output:
 cluster-version: 2.9.0
 runai-public: 
   version: 2.9.0
-  runaiConfigStatus: # (1)
+  runaiConfigStatus:
     conditions:
-    - type: DependenciesFulfilled
+    - type: DependenciesFulfilled   # (1)
       status: "True"
       reason: dependencies_fulfilled
       message: Dependencies are fulfilled
-    - type: Deployed
+    - type: Deployed               
       status: "True"
       reason: deployed
       message: Resources Deployed
     - type: Available
       status: "True"
       reason: available
       message: System Available
-    - type: Reconciled
+    - type: Reconciled              # (2)
       status: "True"
       reason: reconciled
       message: Reconciliation completed successfully
-  optional:  # (2)
-    knative:    # (3)  
+  optional:  # (3)
+    knative:    # (4)  
       components:
         hpa:
           available: true
         knative:
           available: true
         kourier:
           available: true
-    mpi:        # (4) 
+    mpi:        # (5) 
       available: true
 ```
 
-1. Verifies that all mandatory dependencies are met: NVIDIA GPU Operator, Prometheus and NGINX controller. 
-2. Checks whether optional product dependencies have been met.
-3. See [Inference prerequisites](cluster-prerequisites.md#inference).
-4. See [distributed training prerequisites](cluster-prerequisites.md#distributed-training).
+1. Verifies that all mandatory dependencies are met: NVIDIA GPU Operator, Prometheus and NGINX controller.
+2. Verifies that all of Run:ai managed resources have been successfully deployed.
+3. Checks whether optional product dependencies have been met.
+4. See [Inference prerequisites](cluster-prerequisites.md#inference).
+5. See [distributed training prerequisites](cluster-prerequisites.md#distributed-training).
 
 For a more extensive verification of cluster health, see [Determining the health of a cluster](../../troubleshooting/cluster-health-check.md).
 
+### Troubleshooting
+#### Dependencies are not fulfilled
+1. Make sure to install the missing dependencies.
+2. If dependencies are installed, make sure that the CRDs of said dependency are installed, and that the version is supported
+3. Make sure there are no necessary adjustments for specific flavors as noted here https://docs.run.ai/v2.15/admin/runai-setup/cluster-setup/cluster-prerequisites
+
+#### Resources not deployed / System Unavailable / Reconciliation Failed
+1. Run Preinstall diagnostic script https://docs.run.ai/v2.15/admin/runai-setup/cluster-setup/cluster-prerequisites/#pre-install-script and check for issues
+2. Run
+```
+   kubectl get pods -n runai
+   kubectl get pods -n monitoring
+```
+and obtain logs from any failing pod `kubectl logs <pod_name>`
```

Comment:
@roir This will render, but will create warnings and errors in the linter. There need to be spaces before and after codeblocks.
```suggestion

```
   kubectl get pods -n runai
   kubectl get pods -n monitoring
```

and obtain logs from any failing pod `kubectl logs <pod_name>`
```